src/istream.cpp

```diff
@@ -7,6 +7,7 @@
 #include <irods/irods_at_scope_exit.hpp>
 #include <irods/replica.hpp>
 
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. There is a call to `printErrorStack` on line 92.

Reply:
I see.